kernel/ovpn-dco/Makefile

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright (C) 2008-2010 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
```

Comment:
You should assign it to yourself or remove it completely. 
Reference:
https://github.com/openwrt/packages/blob/19596a013cc6f3c161f96ce34f48466fe8ddd99d/CONTRIBUTING.md#makefile-contents-should-contain